README.md

```diff
@@ -343,7 +339,7 @@ export class CreateUsers implements SeedsInterface {
     public async seed(factory: FactoryInterface): Promise<any> {
         await factory
             .get(User, 'admin')
-            .create(1);
+            .create();
```

Comment:
It would be nice when we could use `create(10)` and `create()` so we don't need an additional method `createMany(x)`.